source/Mission.cpp

```diff
@@ -391,6 +411,20 @@ void Mission::Save(DataWriter &out, const string &tag) const
 			out.Write("stealth");
 		if(!isVisible)
 			out.Write("invisible");
+		auto saveColor = [&out](const ExclusiveItem<Color> &color, string tokenName) noexcept -> void {
+			if(!color->IsLoaded())
+				return;
+			if(!color->Name().empty())
+				out.Write("color", tokenName, color->Name());
+			else
+			{
+				const float *rgba = color->Get();
+				out.Write("color", tokenName, rgba[0], rgba[1], rgba[2], rgba[3]);
```

Comment:
We shouldn't save the alpha if we ignore it when loading.
```suggestion
				out.Write("color", tokenName, rgba[0], rgba[1], rgba[2]);
```